src/get-start-and-end-commands.js

```diff
@@ -255,19 +256,31 @@ function createProject(runEmber) {
   };
 }
 
+/**
+ * Install packages from package.json then install the specified package and generate blueprint
+ *
+ * @param {string} projectRoot - Used as cwd for running npm commands as well directory to make file modifications
+ * @param {object} blueprint - Expected to have `packageName`, `version`, and `path` (nullable) attributes
+ * @param {string} packageManager - Expected to be either `npm` or `yarn`
+ * @returns {Promise<void>}
+ */
 module.exports.installAddonBlueprint = async function installAddonBlueprint({
   projectRoot,
-  blueprint
+  blueprint,
+  packageManager = 'npm'
 }) {
   // `not found: ember` without this
-  await spawn('npm', ['install'], { cwd: projectRoot });
+  await spawn(packageManager, ['install'], { cwd: projectRoot });
 
-  let { ps } = await emberInstallAddon({
+  let { ps } = await installAndGenerateBlueprint({
     cwd: projectRoot,
     packageName: blueprint.packageName,
     version: blueprint.version,
     blueprintPath: blueprint.path,
-    stdin: 'pipe'
+    blueprintName: blueprint.name,
+    blueprintOptions: blueprint.options || [],
```

Comment:
I think `blueprint.options` is always an array.

Reply:
removed the default `|| []`